src/warnet/control.py

```diff
@@ -105,43 +117,53 @@ def list_active_scenarios():
     table.add_column("Name", style="cyan")
     table.add_column("Status", style="green")
 
-    for commander in commanders:
-        table.add_row(commander.metadata.name, commander.status.phase.lower())
+    for scenario in active_scenarios:
+        table.add_row(scenario, "deployed")
 
     console.print(table)
 
 
 @click.command()
 def down():
-    """Bring down a running warnet"""
-    with console.status("[bold yellow]Bringing down the warnet...[/bold yellow]"):
-        # Delete warnet-logging namespace
-        if delete_namespace("warnet-logging"):
-            console.print("[green]Warnet logging deleted[/green]")
-        else:
-            console.print("[red]Warnet logging NOT deleted[/red]")
-
-    # Uninstall tanks
-    tanks = get_mission("tank")
-    with console.status("[yellow]Uninstalling tanks...[/yellow]"):
-        for tank in tanks:
-            cmd = f"helm uninstall {tank.metadata.name} --namespace {get_default_namespace()}"
-            if stream_command(cmd):
-                console.print(f"[green]Uninstalled tank: {tank.metadata.name}[/green]")
-            else:
-                console.print(f"[red]Failed to uninstall tank: {tank.metadata.name}[/red]")
-
-    # Clean up scenarios and other pods
-    pods = get_pods()
-    with console.status("[yellow]Cleaning up remaining pods...[/yellow]"):
+    """Bring down a running warnet quickly"""
+    console.print("[bold yellow]Bringing down the warnet...[/bold yellow]")
+
+    namespaces = [get_default_namespace(), "warnet-logging"]
+
+    def uninstall_release(namespace, release_name):
+        cmd = f"helm uninstall {release_name} --namespace {namespace} --wait=false"
+        subprocess.Popen(cmd, shell=True, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        return f"Initiated uninstall for: {release_name} in namespace {namespace}"
+
+    def delete_pod(pod_name, namespace):
+        cmd = f"kubectl delete pod --ignore-not-found=true {pod_name} -n {namespace} --grace-period=0 --force"
+        subprocess.Popen(cmd, shell=True, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        return f"Initiated deletion of pod: {pod_name} in namespace {namespace}"
+
+    with ThreadPoolExecutor(max_workers=10) as executor:
```

Comment:
Can we use this for `deploy()` as well? a 100 tank network takes a lot longer to start now than it did before helm. I think currently its a foreach/stream_command() but before helm all tanks were started in parallel

Reply:
Sure, I don't see why not.